WlihaHackEviction/Controllers/PermitController.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.Common;
+using System.Data.SqlClient;
+using Microsoft.AspNetCore.Mvc;
+using Serenity.Data;
+using WlihaHackEviction.Models;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace WlihaHackPermit.Controllers
+{
+    [Route("api/[controller]")]
+    public class PermitController : Controller
+    {
+        public PermitController()
+        {
+        }
+
+        // GET: api/Permit
+        [HttpGet]
+        public List<TenantInfo> GetTenantsEvictedWithoutPermit()
+        {
+            string query = @"SELECT *
+            FROM [dbo].[TenantsInfo]
+            WHERE [dbo].[TenantsInfo].[Id] =
+            (SELECT [dbo].[EvictionInfo].[TenantId]
+            FROM [dbo].[EvictionInfo]
+            WHERE 
+	        NOT EXISTS(
+	        SELECT* 
+	        FROM [dbo].[AddressInfo]
+	        LEFT JOIN [dbo].[PermitInfo]
+	        ON [dbo].[AddressInfo].[StreetAddress] like [dbo].[PermitInfo].[OriginalAddress1]))";

```

Comment:
looks like like is case sensitive. Let's do LOWER([dbo].[AddressInfo].[StreetAddress]) like LOWER([dbo].[PermitInfo].[OriginalAddress1])

Reply:
let's also check that zip code matches